easybuild/easyblocks/n/nwchem.py

```diff
@@ -199,6 +208,9 @@ def configure_step(self):
             libmpi = "-lmpichf90 -lmpich -lopa -lmpl -lrt -lpthread"
         else:
             raise EasyBuildError("Don't know how to set LIBMPI for %s", mpi_family)
+        if self.cfg['armci_network'] in ["OPENIB"]:
+            libmpi += " -libumad -libverbs -lpthread"
+        #if LooseVersion(self.version) <= LooseVersion("6.5"):
```

Comment:
confirmed, no need to set these environment variables anymore with NWChem 6.6, see https://github.com/akesandgren/easybuild-easyblocks/pull/3